### DISCUSSION TITLE
How to change the default blue color when we click on menu component?

### DISCUSSION THREAD

**Discussion by nishu-murmu** [saadeghi/daisyui#1789] [Q&A] [answered]
How to change the default blue color when we click on menu component?
I was looking for it in the docs? Couldn't find anything yet.

**Comment by saadeghi** [comment, marked as answer]
It's the primary color of current theme:  
- https://daisyui.com/docs/colors/
- https://daisyui.com/docs/themes/

You can create your own themes with custom colors here: https://daisyui.com/theme-generator/
